packages/modules/src/modules/banners/utils/storybook.ts

```diff
@@ -18,7 +18,7 @@ export const tracking: Tracking = {
     campaignCode: 'UsEoyAppealBanner_control',
 };
 
-export const content = {
+export const contentContributions = {
```

Comment:
I reckon we might need to revert this name change, this `content` variable is mainly used to give pass default values to our stories. And it would be better to move `contentCharityAppeal` below to the charity banner story